Makefile

```diff
@@ -126,6 +141,22 @@ cuda-jupyter-tensorflow-ubi9-python-3.9: cuda-jupyter-datascience-ubi9-python-3.
 jupyter-trustyai-ubi9-python-3.9: jupyter-datascience-ubi9-python-3.9
 	$(call image,$@,jupyter/trustyai/ubi9-python-3.9,$<)
 
+# Build and push runtime-datascience-ubi9-python-3.9 image to the registry
+.PHONY: runtime-datascience-ubi9-python-3.9
+runtime-datascience-ubi9-python-3.9: base-ubi9-python-3.9
+	$(call image,$@,runtimes/datascience/ubi9-python-3.9,$<)
+
+# Build and push runtime-pytorch-ubi9-python-3.9 image to the registry
+.PHONY: runtime-cuda-pytorch-ubi9-python-3.9
```

Comment:
The PyTorch notebook is derived from the data science, not from CUDA. So, as base image should have `runtime-datascience-runtime-datascience-ubi9-python-3.9` and the name shall be like: runtime-pytorch-ubi9-python-3.9.

Reply:
as explained in above comment, runtime image just need to be based out of base image.
Doesnt need to dependent as of now.